dex-auth.go

```diff
@@ -50,6 +52,7 @@ func (cluster *Cluster) handleCallback(w http.ResponseWriter, r *http.Request) {
 	var (
 		err   error
 		token *oauth2.Token
+		IDP_CA_PEM string
```

Comment:
from golang style on [initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms), I think this would be IDPCAPEM, which is a bit of a mouthful. There's also the [mixed-caps](https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps) standard which doesn't seem to always be followed in this repo